packaging/Microsoft.Dotnet.Wpf.ProjectTemplates/content/WpfClassLibrary-VisualBasic/Company.ClassLibrary1.vbproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
Shouldn't this be Microsoft.NET.Sdk.WindowsDesktop as well?

Reply:
fixing